azure_functions/src/functions/caDphTimerTrigger.ts

```diff
@@ -15,27 +15,22 @@ export async function caDphTimerTrigger(myTimer: Timer, context: InvocationConte
     console.log("Context:")
     console.log(context);
     // context.extraInputs.get("customer")
-    // TODO - send a real message
-
-    // TODO - Adjust the visibility timeout to be a fairly high value.  We want to avoid enqueuing the message and then having the
-    // polling handler kick off to download it but the download takes longer than the timeout, this would in theory kick off an infinite loop
-    // that would lock out our account
 
     // We set the visibility timeout for the message on reading, in queue.go
     // messageTimeToLive of -1 means the message does not expire
-    // const sendMessageResponse = await queueClient.sendMessage("cheezburger", {messageTimeToLive: -1})
-    // console.log("Sent message successfully, service assigned message Id:", sendMessageResponse.messageId, "service assigned request Id:", sendMessageResponse.requestId );
+    // the queue message contents will (in future) be the key to client-specific config
+    const sendMessageResponse = await queueClient.sendMessage("cadph", {messageTimeToLive: -1})
+    console.log("Sent message successfully, service assigned message Id:", sendMessageResponse.messageId, "service assigned request Id:", sendMessageResponse.requestId );
 
     context.log('Timer function processed request.');
 }
 // TODO - set up the right CRON expression
-// TODO - figure out how we make sure there's only one Azure Function running - we should alarm if there's more than one
 // TODO - figure out if we can add multiple timers (like one per external customer?) to the same function
 // TODO - find out the timer's timezone for scheduling - there's a `schedule: { adjustForDST: true }` setting in the timer
 // TODO - Azure SDK Timeout setup research for when calling azure doesn't work.
 // https://learn.microsoft.com/en-us/azure/azure-functions/functions-bindings-timer?tabs=python-v2%2Cisolated-process%2Cnodejs-v4&pivots=programming-language-typescript#ncrontab-time-zones
 app.timer('caDphTimerTrigger', {
-    schedule: '0 */1 * * * *',
+    schedule: caDphPollingCron, // 9:30 am every monday in january
```

Comment:
Should this comment be moved to where the variable is actually set?

Reply:
I like this suggestion.  We should do this.